docs/freqai-configuration.md

```diff
@@ -0,0 +1,283 @@
+                    .mean()
+                    / df["close"]
+                    - 1
+                    )
+    ```
+
+    (Please see the example script located in `freqtrade/templates/FreqaiExampleStrategy.py` for a full example of `populate_any_indicators()`.)
+
+*Important*: The `self.freqai.start()` function cannot be called outside the `populate_indicators()`.
+
+## Setting the `startup_candle_count`
+Users need to take care to set the `startup_candle_count` in their strategy the same way they would for any normal Freqtrade strategy (see details [here](strategy-customization.md#strategy-startup-period)). This value is used by Freqtrade to ensure that a sufficient amount of data is provided when calling on the `dataprovider` to avoid any NaNs at the beginning of the first training. Users can easily set this value by identifying the longest period (in candle units) that they pass to their indicator creation functions (e.g. talib functions). In the present example, the user would pass 20 to as this value (since it is the maximum value in their `indicators_periods_candles`).
+
+!!! Note
+    Typically it is best for users to be safe and multiply their expected `startup_candle_count` by 2. There are instances where the talib functions actually require more data than just the passed `period`. Anecdotally, multiplying the `startup_candle_count` by 2 always leads to a fully NaN free training dataset. Look out for this log message to confirm that your data is clean:
+
+    ```
+    2022-08-31 15:14:04 - freqtrade.freqai.data_kitchen - INFO - dropped 0 training points due to NaNs in populated dataset 4319.
+    ```
+
+## Creating a dynamic target threshold
+
+The `&*_std/mean` return values describe the statistical fit of the user defined label *during the most recent training*. This value allows the user to know the rarity of a given prediction. For example, `templates/FreqaiExampleStrategy.py`, creates a `target_roi` which is based on filtering out predictions that are below a given z-score of 1.25.
+
+```python
+dataframe["target_roi"] = dataframe["&-s_close_mean"] + dataframe["&-s_close_std"] * 1.25
+dataframe["sell_roi"] = dataframe["&-s_close_mean"] - dataframe["&-s_close_std"] * 1.25
+```
+
+If the user wishes to consider the population
+of *historical predictions* for creating the dynamic target instead of the trained labels, (as discussed above) the user
+can do so by setting `fit_live_prediction_candles` in the config to the number of historical prediction candles
+the user wishes to use to generate target statistics.
+
+```json
+    "freqai": {
+        "fit_live_prediction_candles": 300,
+    }
+```
+
+If the user sets this value, FreqAI will initially use the predictions from the training data
+and subsequently begin introducing real prediction data as it is generated. FreqAI will save
+this historical data to be reloaded if the user stops and restarts a model with the same `identifier`.
+
+
+## Parameter table
```

Comment:
I think this document jumps around a bit

we start with configuration - but don't explain options.
then we go to the strategy - but return to the parameters table down here again.

maybe either moving the parameters table up (or the top section down here) - or splitting this document completely into "configuration" and "strategy" might make more sense? (just a thought though)

Reply:
I agree 100%

Reply:
We went a different direction . I think users need to be introduced to the config before being hit with a lengthy parameter table. The config is quite readable by design (most parameters have verbs and units). 

Now the param table is its own page - which is easily seen from variohs "heights" in FT doc.